modules/claims_api/spec/fixtures/v2/veterans/disability_compensation/form_526_json_api.json

```diff
@@ -70,18 +70,18 @@
             "disabilities": [
                 {
                     "disabilityActionType": "REOPEN",
-                    "name": "Musculoskeletal - Foot",
-                    "classificationCode": "8994",
+                    "name": "Traumatic Brain Injury",
```

Comment:
are changes to this needed for this PR? this is just because we used a different Claim, right? I feel like these are irrelevant to this PR

Reply:
Yes these changes are needed based on recent changes to validation, without them the request and the tests will fail.